extras_install.sh

```diff
@@ -26,12 +26,12 @@ ID=$(cat /etc/os-release | grep -w ID | cut -f2 -d"=")
 # https://github.com/google-coral/examples-camera/blob/master/opencv/install_requirements.sh
 
 if [ ${ID} = "raspbian" ]; then
-   pip3 install opencv-contrib-python==4.1.0.25
+   python3 -m pip  install opencv-contrib-python==4.1.0.25
 fi
 
 if [ ${ID} = "ubuntu" ]; then
-   pip3 install --upgrade pip
-   pip3 install opencv-contrib-python
+   python3 -m pip  install --upgrade pip
+   python3 -m pip  install opencv-contrib-python==4.6.0.66
```

Comment:
Did you test with different python versions `<=3.6 & >=3.6`?
It's better to test against ub18 & ub20 platforms. On fresh instance & on upgraded/existing instance.